src/js/inverted-index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * InvertedIndex class
+ * @class
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.indexes = {};
```

Comment:
If this is supposed to store all the indices created why isn't it declared as an array?